docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -0,0 +1,327 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6aa68656",
+   "metadata": {},
+   "source": [
+    "# Variational Quantum Eigensolver with Estimator Primitive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15a4588c",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "The Variational Quantum Eigensolver (VQE) is an optimization routine for finding the ground state energy (i.e. lowest eigenvalue) of a Hamiltonian and is a considered to be a viable candidate for NISQ hardware. In this tutorial, we will go over how to use Qiskit Runtime to submit variational jobs using the estimator. Specifically, we will be looking at calculating the ground state energy of the $H_2$ molecule using the estimator primitive. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5cb8630",
+   "metadata": {},
+   "source": [
+    "### Create Service\n",
+    "First we have to create our service instance and specify our backend. In this example we will be working with a simulator for the sake of speed, but by simply changing this one line (for choice of backend) this could run on hardware as well. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9c482297",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import QiskitRuntimeService, Estimator\n",
+    "\n",
+    "service = QiskitRuntimeService()\n",
+    "\n",
+    "options = {\"backend\" : \"ibmq_qasm_simulator\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e78c4ba0",
+   "metadata": {},
+   "source": [
+    "### Molecular Hamiltonians\n",
+    "\n",
+    "We will be working with Qiskit Nature to generate and handle molecular hamiltonians. First though, we need to install these packages. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a635d559",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install qiskit-nature\n",
+    "pip install 'qiskit-nature[pyscf]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d722d9ac",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Now we need to generate the Hamiltonians that we wish to find the ground state energy of. For this task we will be utilizing qiskit nature. First we have to specify how we are converting the fermionic operators of the electronic molecular hamiltonian to qubit operators. Here we will use the Parity Mapper, which maps annihilation to Pauli operators via $ \\hat{a} \\rightarrow \\frac{1}{2} \\left ( X_p Z_{p-1} + i Y_p \\right ) X_{p+1} ... X_{N} $"
```

Comment:
```suggestion
    "Now we need to generate the Hamiltonians that we wish to find the ground state energy of. For this task we will be utilizing qiskit nature. First we have to specify how we are converting the fermionic operators of the electronic molecular hamiltonian to qubit operators. Here we will use the Parity Mapper, which maps annihilation to Pauli operators via $\\hat{a} \\rightarrow \\frac{1}{2} \\left ( X_p Z_{p-1} + i Y_p \\right ) X_{p+1} ... X_{N}$"
```
Doc build is failing due to this, doesn't like the leading and trailing space in the formula.

Reply:
There's also another later in the notebook